packages/dht/src/connection/webrtc/BrowserWebrtcConnection.ts

```diff
@@ -65,15 +65,15 @@ export class NodeWebrtcConnection extends EventEmitter<Events> implements IWebrt
                         this.makingOffer = true
                         try {
                             await this.peerConnection.setLocalDescription()
-                        } catch (err) {
-                            logger.warn('error', { err })
+                        } catch (error) {
+                            logger.warn('error', { error })
```

Comment:
```suggestion
                            logger.warn('Failed to set local description', { err })
```

It is good to keep the metadata key as err, because Pino hides the error details if we rename it to error.
E.g.:
```ts
const error = new Error('Foo')
logger.warn('Bar', { err: error })
logger.warn('Bar', { error })
```

```
WARN [0000-00-00T00:00:00.000] (.                        ): Bar
    err: {
      "type": "Error",
      "message": "Foo",
      "stack":
          Error: Foo
              at main (...)
              at Object.<anonymous> (...)
              ...
    }              
WARN [0000-00-00T00:00:00.000] (.                        ): Bar
    error: {}
```